src/pages/home/HeaderView.js

```diff
@@ -39,51 +45,66 @@ const defaultProps = {
     report: null,
 };
 
-const HeaderView = props => (
-    <View style={[styles.appContentHeader]} nativeID="drag-area">
-        <View style={[styles.appContentHeaderTitle, !props.isSmallScreenWidth && styles.pl5]}>
-            {props.isSmallScreenWidth && (
-                <Pressable
-                    onPress={props.onNavigationMenuButtonClicked}
-                    style={[styles.LHNToggle]}
-                >
-                    <Icon src={BackArrow} />
-                </Pressable>
-            )}
-            {props.report && props.report.reportName ? (
-                <View
-                    style={[
-                        styles.flex1,
-                        styles.flexRow,
-                        styles.alignItemsCenter,
-                        styles.justifyContentBetween,
-                    ]}
-                >
+const HeaderView = (props) => {
+    const participants = lodashGet(props.report, 'participants', []);
```

Comment:
```suggestion
    const participants = props.report.participants || [];
```

Reply:
The report is not present immediately here so we have to use it like  `props.report && props.report.participants` but I think we use `Lodashget` for this purpose.